tftrt/examples/object_detection/object_detection.py

```diff
@@ -534,7 +534,9 @@ def benchmark_model(frozen_graph,
         raise RuntimeError(
             'Fixed image shape must be provided for batch size > 1')
 
-    if not use_synthetic:
+    if use_synthetic:
```

Comment:
Instead of this, could you fix the problem at https://github.com/tensorflow/tensorrt/blob/master/tftrt/examples/object_detection/object_detection.py#L614

`(len(image_ids) + batch_size - 1) / batch_size,` --> `(num_images + batch_size - 1) / batch_size,`